tests/system/action/projector/test_next.py

```diff
@@ -0,0 +1,53 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class ProjectorNext(BaseActionTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.set_models(
+            {
+                "meeting/1": {},
+                "projector/2": {"meeting_id": 1},
+                "projector/3": {
+                    "current_projection_ids": [1, 2],
+                    "preview_projection_ids": [3, 4],
+                    "history_projection_ids": [],
+                    "meeting_id": 1,
+                },
+                "projector/4": {
+                    "current_projection_ids": [],
+                    "preview_projection_ids": [5],
+                    "history_projection_ids": [],
+                    "meeting_id": 1,
+                },
+                "projection/1": {"current_projector_id": 3, "meeting_id": 1},
+                "projection/2": {"current_projector_id": 3, "meeting_id": 1},
+                "projection/3": {"preview_projector_id": 3, "meeting_id": 1},
+                "projection/4": {"preview_projector_id": 3, "meeting_id": 1},
+                "projection/5": {"preview_projector_id": 4, "meeting_id": 1},
+            }
+        )
+
+    def test_next_nothing(self) -> None:
+        response = self.request("projector.next", {"id": 2})
+        self.assert_status_code(response, 200)
+        projection = self.get_model("projector/2")
```

Comment:
`projection` -> `projector` (In all tests)

Reply:
Yes, fix that typos.